examples/Makefile

```diff
@@ -0,0 +1,10 @@
+all: kapitan status
+
+.PHONY: kapitan
+kapitan:
```

Comment:
probably better to rename this to `compile` instead of `kapitan`

Reply:
done